index.js

```diff
@@ -67,18 +67,20 @@ export default class RNUrlPreview extends React.PureComponent {
     Linking.openURL(this.props.text.match(REGEX)[0]);
   };
 
-  renderImage = (imageLink, faviconLink, imageStyle, faviconStyle) => {
+  renderImage = (
+    imageLink,
+    faviconLink,
+    imageStyle,
+    faviconStyle,
+    imageProps = { resizeMode: "contain" }
```

Comment:
i think imageProps default value should be placed in RNUrlPreview.defaultProps instead of passing it here